src/solvers/refinement/bv_refinement.h

```diff
@@ -15,6 +15,7 @@ Author: Daniel Kroening, kroening@kroening.com
 #include <util/ui_message.h>
 
 #include <solvers/flattening/bv_pointers.h>
+#include <solvers/sat/satcheck_minisat2.h>
```

Comment:
This implies that no solver other than Minisat2 could be used. This is certainly a blocker.

Reply:
Suggested move of constant definitions to the constant file should alleviate that.

Reply:
Yep; that's a blocker for several other industrial users.